renderer/tdf/common/include/renderer/tdf/tdf_render_manager.h

```diff
@@ -25,6 +25,7 @@
 #include "dom/dom_node.h"
 #include "dom/render_manager.h"
 #include "footstone/serializer.h"
+#include "vfs/uri_loader.h"
```

Comment:
missing blank line, between `#include` and `#pragma`

Reply:
fixed